docs/en/observability/apm/tab-widgets/install-agents.asciidoc

```diff
@@ -223,91 +217,74 @@ Read more in the {apm-ios-ref}/intro.html[APM iOS Agent Reference].
 
 // tag::java[]
 
+There are three different ways to set up the Elastic APM Java Agent:
+
+. **Manual setup with `-javaagent` flag** +
+  Manually set up and configure the agent with the `-javaagent` JVM option. No application code change required, requires
+application restart. See below for more information on this setup method.
+. **Automatic setup with `apm-agent-attach-cli.jar`** +
+  Automatically set up the agent without needing to alter the configuration of your JVM or application server. No application code
+  nor JVM options changes required, allows attaching to a running JVM. Refer to the {apm-java-ref-v}/setup-attach-cli.html[Java agent documentation] for more information on this setup method.
+. **Programmatic API setup to self-attach** +
+  Set up the agent with a one-line code change and an extra `apm-agent-attach` dependency. No modification of JVM options,
+  the agent artifact is embedded within the packaged application binary. Refer to the {apm-java-ref-v}/setup-attach-api.html[Java agent documentation] for more information on this setup method.
+
```

Comment:
I'm confused about how these three options relate to the rest of the content for the Java agent.

Also if these three options are not sequential steps, I would recommend an unordered list instead of an ordered one.

Reply:
Yeah, fair point. Previously, we only displayed the `-javaagent` method of setting up the Java agent. There is no recommended path to set up the agent, so I attempted to briefly explain all three before diving into one. I'll try moving this section down and see what that looks like.

Reply:
Fixed in b4f3240